plugins/executor.js

```diff
@@ -11,7 +11,8 @@ const SCHEMA_START = Joi.object().keys({
     apiUri: Joi.string().uri().required()
         .label('API URI'),
     token: Joi.string().required()
-        .label('Build JWT')
+        .label('Build JWT'),
+    tolerations: Joi.array().optional()
```

Comment:
We shouldn't need this change because of my comment here: https://github.com/screwdriver-cd/executor-k8s-vm/pull/19#pullrequestreview-85743527

Reply:
I think this is required since, `executor-base` does schema validation.

Reply:
this schema is for this: https://github.com/screwdriver-cd/executor-k8s-vm/blob/master/index.js#L71-L80

We are not changing the method signature for `start`, so there is no need to change this schema.